rasa/utils/endpoints.py

```diff
@@ -1,10 +1,24 @@
 import aiohttp
 from typing import Any, Optional, Text, Dict
 
+from sanic.request import Request
+
 import rasa.utils.io
 from rasa.constants import DEFAULT_REQUEST_TIMEOUT
 
 
+def default_arg(request: Request, key: Text, default: Any = None) -> Optional[Any]:
+    """Return an argument of the request or a default.
+
+    Checks the `name` parameter of the request if it contains a value.
+    If not, `default` is returned."""
+    found = request.raw_args.get(key)
+    if found is not None:
```

Comment:
`return found or default`

Reply:
Nope, forget this comment. Guess that's to dangerous in case values are `0` or `[]`